plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/internal/OutputFile.kt

```diff
@@ -0,0 +1,44 @@
+package de.fayard.refreshVersions.internal
+
+import org.gradle.api.Project
+
+internal enum class OutputFile(var path: String, var existed: Boolean = false, val alternativePath: String? = null) {
+    OUTPUTDIR("buildSrc/src/main/kotlin"),
```

Comment:
Would probably be better renamed to `OUTPUT_DIR`.

Reply:
done